docs/compute/compiling.md

```diff
@@ -15,7 +15,7 @@ module load hdf5
 
```

Comment:
On this page there is usage of module version that are not on our system. Please go through this page and change these so that it reflects the current state of Alpine.

Reply:
It looks like you addressed all of these except for the output in the section `Compiling and Linking` for the part `[johndoe@@c3cpu-a5-u17-2 ~]$ env | grep CURC`